src/test/java/seedu/address/logic/commands/RemarkCommandTest.java

```diff
@@ -36,7 +36,7 @@
     @Test
     public void execute_addRemarkUnfilteredList_success() throws Exception {
         Person firstPerson = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        Person editedPerson = new PersonBuilder(firstPerson).withRemark(REMARK_EXAMPLE).buildRegardlessOfType();
+        Person editedPerson = new PersonBuilder(firstPerson).withRemark(REMARK_EXAMPLE).build();
```

Comment:
In these test cases, we should test for a similar situation as we just encountered; as in, remarking on a Lead or Contact won't change it to the other type.

Right now, you've only tested on FIRST_PERSON, who I think is a Lead. Please check a Contact too.